### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by rekhakilari**
- [DEMO LINK](https://rekhakilari.github.io/layout_moyo-header-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_moyo-header-en/report/html_report/)

### CODE REVIEW

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_moyo-header/report/html_report/)

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_moyo-header-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_moyo-header-en/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,98 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+         src="./images/logo.png"
+         alt="company logo"
+        >
+      </div>
```
Comment (VinnieJ-2k20) [inline src/index.html:22]: This element should be the link to the home page.
image

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,98 @@
...
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+             class="
+                nav__link
+                is-active
+                is-active1"
+                href="#"
```
Comment (VinnieJ-2k20) [inline src/index.html:32]: Don't break lines in class.
```suggestion
             class="nav__link is-active is-active1"
             href="#"
```

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,98 @@
...
+            >
+             smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+               data-qa="hover"
+               href="#"
+            >
```
Comment (VinnieJ-2k20) [inline src/index.html:60]: Adjust the code style here.
```suggestion
            <a
              class="nav__link"
              data-qa="hover"
              href="#"
            >
```

File: src/index.html [hunk 4]
```diff
@@ -6,9 +6,98 @@
...
+             href="#"
+            >
+             gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link"
+             href="#"
+            >
```
Comment (VinnieJ-2k20) [inline src/index.html:77]: Same here
```suggestion
            <a
              class="nav__link"
              href="#"
            >
```

File: src/index.html [hunk 5]
```diff
@@ -6,9 +6,98 @@
...
+          <li class="nav__item">
+            <a
+             class="nav__link"
+             href="#"
+            >
+             photo
+            </a>
+          </li>
+
+          <li class="nav__item item-last">
```
Comment (VinnieJ-2k20) [inline src/index.html:91]: You don't use this class in styles, remove it.
```suggestion
          <li class="nav__item">
```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,71 @@
...
+}
+
+.nav__link {
+  height: 60px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  cursor: pointer;
```
Comment (VinnieJ-2k20) [inline src/style.css:53]: `cursor: pointer;` is default for the link. This rule here is redundant.
```suggestion
```

File: src/style.css [hunk 7]
```diff
@@ -1 +1,71 @@
...
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-51%);
```
Comment (VinnieJ-2k20) [inline src/style.css:66]: Use left and right coordinates to position this pseudo-element.

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
You are on the right way.
But some changes to code and code style are needed.

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_stars/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_stars/report/html_report/)

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekhakilari.github.io/layout_moyo-header-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_moyo-header-en/report/html_report/)

**Pull request closed without merging**